graphql/src/main/java/com/linecorp/armeria/common/graphql/GraphqlUtil.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.graphql;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Map;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import graphql.GraphQLContext;
+import graphql.com.google.common.collect.ImmutableMap;
+import graphql.schema.DataFetchingEnvironment;
+
+/**
+ * Utility for handling the GraphQL.
+ */
+@UnstableApi
+public final class GraphqlUtil {
+
+    private static final String GRAPHQL_CONTEXT_KEY = "com.linecorp.armeria.graphql.context.key";
+
+    /**
+     * Returns a {@link Map} containing the {@link ServiceRequestContext}.
+     */
+    public static Map<String, Object> graphqlContext(ServiceRequestContext requestContext) {
```

Comment:
At second thought, I feel like we need to move this class to `server.graphql` because it's for the server side.

Reply:
I think so too. 🤣